geminimd/commands.py

```
@@ -0,0 +1,401 @@
+    totals.setdefault("attachments", 0)
+    return SyncResult(
+        count=len(items),
+        output_dir=options.output_dir,
+        folder_name=options.folder_name,
+        folder_id=folder_id,
+        items=items,
+        total_stats=dict(totals),
+    )
+
+
+def status_command(env: CommandEnv) -> StatusResult:
```

Ruff (0.15.9)

[warning] Unused function argument: `env`

(ARG001)